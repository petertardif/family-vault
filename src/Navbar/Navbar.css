
.Navbar {
  background-color: #101732;
  position: fixed;
  top: 0;
  left: 0;
  right: 0;
  z-index: 10;
  display: flex;
  justify-content: space-between;
  padding: 10px 30px;
  box-shadow: 0px 1.5px 3px #000000;
}

.Navbar__Icon > a {
  color: #c50824;
  font-family: 'Baloo', cursive;
  font-size: 40px;
  margin: 0 auto; 
  text-decoration: none;
}

.Navbar__Listitems > a {
  color: #fefefe;
  text-decoration: none;
}

.Navbar__Listitems > a:hover {
  color: rgba(197, 8, 36, 0.9);
  text-decoration: none;
}
