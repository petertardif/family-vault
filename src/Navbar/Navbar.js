import React, { Component } from 'react';
import { Link } from 'react-router-dom';
import Button from '../Button/Button';

export default class Navbar extends Component {
  render() {
    return (
      <>
        <nav role="navigation">
          <div>
            <h1>FV</h1>
          </div>
          <ul>
            {/* <li><button id="myBtn">Sign up / Log in</button></li> */}
            <li><Button tag={Link} to='/userlanding' type='button' className='Navbar_tour-button'
          >Demo</Button></li>
          </ul>
        </nav>
      </>
    )
  }
}
