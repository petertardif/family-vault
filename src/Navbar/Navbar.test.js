import React from 'react';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Navbar from './Navbar'

describe(`Navbar component`, () => {

  it('renders a .Navbar by default', () => {
    const wrapper = shallow(<Navbar />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})