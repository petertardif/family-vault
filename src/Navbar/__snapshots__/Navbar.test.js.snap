// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar component renders a .Navbar by default 1`] = `
<Fragment>
  <nav
    role="navigation"
  >
    <div>
      <h1>
        FV
      </h1>
    </div>
    <ul>
      <li>
        <button
          id="myBtn"
        >
          Sign up / Log in
        </button>
      </li>
      <li>
        <Button
          className="Navbar_tour-button"
          tag={[Function]}
          to="/userlanding"
          type="button"
        >
          Demo
        </Button>
      </li>
    </ul>
  </nav>
</Fragment>
`;
